pages/[prefix]/explore/items.vue

```diff
@@ -46,12 +46,18 @@ definePageMeta({
   layout: 'explore-layout',
   middleware: [
     function (to) {
-      if (to.query.listed === undefined) {
+      const extraQuery = ['listed', 'art_gen'].reduce((acc, key) => {
```

Comment:
will break kodadot

Reply:
It will not break other domains because it will finally use `useArtGenMode` to check if the domain was matched.

Reply:
Ah ok, missed that part